lib/config-generator.js

```diff
@@ -302,7 +302,7 @@ class ConfigGenerator {
 
             rules.push(applyRuleConfigurationCallback('images', {
                 test: /\.(png|jpg|jpeg|gif|ico|svg|webp)$/,
-                loader: loaderName,
+                loader: require.resolve(loaderName),
```

Comment:
This is a nit-pick, but I'm asking in case it's important. `loaderName` here may be `file-loader` or `url-loader`. `file-loader` is required by Encore, but `url-loader` is something that the user must have installed. What affect does `require.resolve()` have on a package that we *know* the user must have installed?

Reply:
That's already what we do to check if those kind of packages are installed:

https://github.com/symfony/webpack-encore/blob/134f47a39eed7c832dd367ccdb5952be06fd7161/lib/package-helper.js#L61-L68